build/main-branch-sync/rotate-secrets.sh

```diff
@@ -14,6 +16,13 @@ for CLI in gh aws jq yq oc; do
   fi
 done
 
+# Verify the oc version
+MAJOR_OC=$(oc version -o json | jq -r '.clientVersion.gitVersion' | grep -o "[0-9]\+\.[0-9]\+" | sed 's/^[0-9]*\.//')
+MINOR_OC=$(oc version -o json | jq -r '.clientVersion.gitVersion' | grep -o "[0-9]\+\.[0-9]\+" | sed 's/\.[0-9]*$//')
```

Comment:
I might be wrong... but I test with 4.11 and sed 's/\.[0-9]*$//' then it returns 4 which is `MINOR_OC`. Is this what you intended?

Reply:
Hmmm. That's odd. No, that's not the intention. What does `oc version -o json` give?

Reply:
```
{
  "clientVersion": {
    "major": "",
    "minor": "",
    "gitVersion": "4.10.0-202212051506.p0.g0c49433.assembly.stream-0c49433",
```

Reply:
I guess MAJOR_OC and MINOR_OC should be changed

Reply:
Oh, shoot. I got them flipped. Thanks for double checking it!